packages/kbn-es/src/utils/docker.ts

```diff
@@ -309,16 +311,18 @@ export function resolveDockerImage({
  * Determine the port to bind the Serverless index node or Docker node to
  */
 export function resolvePort(options: ServerlessOptions | DockerOptions) {
+  const port = options.port || DEFAULT_PORT;
+  const value = ['-p', `127.0.0.1:${port}:${port}`];
+
+  if ((options as ServerlessOptions).host) {
+    value.push('-p', `${(options as ServerlessOptions).host}:${port}:${port}`);
+  }
```

Comment:
Just curious if this change needed for Cypress tests or as a general capability to improve developer UX?
If we are keeping the function name `resolvePort` as is, could you update the description to have few words about `host` usage

Reply:
this change is needed to improve dev UX, mostly in case we want to set elastic-agent on a dedicated VM outside of the Docker network

Reply:
@dmlemeshko updated description, please let me know if that works